pkg/cmd/registry/setting/get/get.go

```diff
@@ -0,0 +1,120 @@
+package get
+
+import (
+	"context"
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/registry/registrycmdutil"
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/registry/rule/rulecmdutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/cmdutil/flagutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/dump"
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/iostreams"
+	"github.com/redhat-developer/app-services-cli/pkg/core/localize"
+	"github.com/redhat-developer/app-services-cli/pkg/core/logging"
+	"github.com/redhat-developer/app-services-cli/pkg/core/servicecontext"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	"github.com/spf13/cobra"
+
+	"github.com/redhat-developer/app-services-cli/pkg/shared/contextutil"
+)
+
+type options struct {
+	IO             *iostreams.IOStreams
+	Connection     factory.ConnectionFunc
+	Logger         logging.Logger
+	localizer      localize.Localizer
+	Context        context.Context
+	ServiceContext servicecontext.IContext
+
+	registryID  string
+	settingName string
+	output      string
+}
+
+// NewGetCommand creates a new command to get a service registry setting
+func NewGetCommand(f *factory.Factory) *cobra.Command {
+
+	opts := &options{
+		IO:             f.IOStreams,
+		Connection:     f.Connection,
+		Logger:         f.Logger,
+		localizer:      f.Localizer,
+		Context:        f.Context,
+		ServiceContext: f.ServiceContext,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "get",
+		Short:   f.Localizer.MustLocalize("setting.get.cmd.description.short"),
+		Long:    f.Localizer.MustLocalize("setting.get.cmd.description.long"),
+		Example: f.Localizer.MustLocalize("setting.get.cmd.example"),
+		Args:    cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, _ []string) (err error) {
+
+			if opts.settingName == "" {
+				if !opts.IO.CanPrompt() {
+					return flagutil.RequiredWhenNonInteractiveError("setting-name")
+				}
+				err = runInteractivePrompt(opts)
+				if err != nil {
+					return err
+				}
+			}
+
+			registryInstance, err := contextutil.GetCurrentRegistryInstance(f)
+			if err != nil {
+				return err
+			}
+
+			opts.registryID = registryInstance.GetId()
```

Comment:
This can be empty. We should replicate the same logic/checks as other commands do

Reply:
Meaing:

Registry id should be passed as flag and when not present in context we should error.

Reply:
Oh, I have not realized that. It should be fixed now.